packages/svelte/tests/runtime-browser/samples/binding-width-height-this-timing/_config.js

```diff
@@ -7,6 +7,7 @@ export default test({
 
 		assert.deepEqual(log, [
 			[false, 0, 0],
+			[true, 0, 0],
```

Comment:
this change is a dealbreaker, I'm afraid

Reply:
Ekk, okay let me take another round at seeing if I can get the effects to fire sooner.